FrontEnd/Movies-Library/src/app/models/AggregatedMovie.ts

```diff
@@ -0,0 +1,11 @@
+export interface AggregatedMovie {
+  status: string;
+  rating?: number;
+  is_favorite: boolean;
+  movie: {
```

Comment:
Instead of creating an object inside the interface. Create a new movie model inside this file.